src/libServer/Server.cpp

```diff
@@ -287,7 +260,10 @@ Json::Value Server::GetTxBlock(const string& blockNum) {
   }
 }
 
-string Server::GetGasPrice() { return "Hello"; }
+string Server::GetMinimumGasPrice() {
+  DSBlock latestDsBlock = m_mediator.m_dsBlockChain.GetLastBlock();
+  return latestDsBlock.GetHeader().GetGasPrice().str();
```

Comment:
How about "return m_mediator.m_dsBlockChain.GetLastBlock().GetHeader().GetGasPrice().str();" to avoid temporarily memory used?

Reply:
ok